gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/grpc.py.j2

```diff
@@ -56,13 +56,19 @@ class MetadataClientInterceptor(grpc.UnaryUnaryClientInterceptor):
     def intercept_unary_unary(self, continuation, client_call_details, request):
         request_metadata = client_call_details.metadata
         if CLIENT_LOGGING_SUPPORTED and _LOGGER.isEnabledFor(logging.DEBUG):
+            grpc_request = {
+              "payload":   type(request).to_json(request),
+              "requestMethod": "grpc",
```

Comment:
I don't think we need this for `gRPC`. `method` and `url` are rest only fields.

Reply:
But how will the user looking at the logs know (from the logs only) that this was a gRPC request? They have to reason from not having a URL or method field, or from looking at the metadata. We might as well make it explicit for convenience.

No need to block on this; we can change it later.